tiledb/CMakeLists.txt

```diff
@@ -1088,7 +1091,10 @@ if (TILEDB_STATIC)
     )
     # Create the target
     add_library(tiledb_static STATIC
-      $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS_STATIC>
+      $<TARGET_OBJECTS:TILEDB_CORE_OBJECTS_STATIC>)
+    target_link_libraries(tiledb_static
+      PRIVATE
```

Comment:
Could it be that you're not getting symbol published because this is PRIVATE and not PUBLIC?